tests/platform_tests/api/test_psu_fans.py

```diff
@@ -75,6 +75,22 @@ def compare_value_with_platform_facts(self, duthost, key, value, psu_idx, fan_id
                           "'{}' value is incorrect. Got '{}', expected '{}' for fan {} within psu {}".format(key, value, expected_value, fan_idx, psu_idx))
 
 
+    def get_fan_facts(self, duthost, psu_idx, fan_idx, def_value, *keys):
```

Comment:
get_psu_facts?

Reply:
@dflynn-Nokia  @ArunSaravananBalachandran could you check this?

Reply:
@Blueve. This method is currently used to retrieve the facts of a given PSU Fan (test_psu_fans.py) and not the complete PSU facts.